srcpkgs/rosegarden/template

```diff
@@ -12,7 +12,10 @@ maintainer="newbluemoon <blaumolch@mailbox.org>"
 license="GPL-2.0-or-later"
 homepage="http://rosegardenmusic.com/"
 distfiles="${SOURCEFORGE_SITE}/rosegarden/rosegarden/${version/*.*.*/${version%.*}}/rosegarden-${version}.tar.bz2"
-checksum=886684afc5858a9578234d1f845188db130114f7fbf38208c4d5ecda15131c5b
+checksum=cde4fde48d01024a03bd7a96f63f9c6e7324728214183ceb605018f89faf0b66
+# check requires a display:
+# qt.qpa.xcb: could not connect to display
```

Comment:
Try adding a `pre_check` like `kparts` does.

Reply:
Yes, that did the trick, thanks. However, one test requires `XDG_RUNTIME_DIR` to be set. So I added it, too, but maybe there is a more elegant solution I just don’t see?

Reply:
We could have it exported by `xbps-src` by default, but I'm not sure that's ideal. Your current fix looks good to me.